web/src/add_subscribers_pill.ts

```diff
@@ -99,6 +99,39 @@ export function create({
     return pill_widget;
 }
 
+export function create_without_add_button({
+    $pill_container,
+    get_potential_subscribers,
+    onPillCreateAction,
+    onPillRemoveAction,
+}: {
+    $pill_container: JQuery;
+    get_potential_subscribers: () => User[];
+    onPillCreateAction: (pill_user_ids: number[]) => void;
+    onPillRemoveAction: (pill_user_ids: number[]) => void;
+}): CombinedPillContainer {
+    const pill_widget = input_pill.create<CombinedPill>({
+        $container: $pill_container,
+        create_item_from_text,
+        get_text_from_item,
+    });
+    function get_users(): User[] {
+        const potential_subscribers = get_potential_subscribers();
+        return user_pill.filter_taken_users(potential_subscribers, pill_widget);
+    }
```

Comment:
This one function is duplicated from above, but don't think it's worth it to have shared code this for a 2 line function

Reply:
Yeah the duplication is a little clunky, but it's not a huge amount of code overall, I think we can leave it as-is.